tests/api/v2/handlers/test_operations_api.py

```diff
@@ -0,0 +1,241 @@
+    test_source = Source(id='123', name='test', facts=[test_fact], adjustments=[])
+    loop.run_until_complete(BaseService.get_service('data_svc').store(test_source))
+
+    expected_operation = {'name': 'My Test Operation',
+                          'adversary': AdversarySchema().dump(test_adversary),
+                          'state': 'paused',
+                          'id': '123',
+                          'group': 'red',
+                          'autonomous': 0,
+                          'planner': PlannerSchema().dump(test_planner),
+                          'source': SourceSchema().dump(test_source),
+                          'jitter': '2/8',
+                          'visibility': 50,
+                          'auto_close': False,
+                          'obfuscator': 'plain-text',
+                          'use_learning_parsers': False}
+
+    test_operation = OperationSchema().load(expected_operation)
+
+    test_agent = Agent(paw='123', sleep_min=2, sleep_max=8, watchdog=0, executors=['sh'], platform='linux')
+    test_operation.agents.append(test_agent)
+    test_operation.set_start_details()
+    loop.run_until_complete(BaseService.get_service('data_svc').store(test_agent))
+    test_executor = ExecutorSchema().load(dict(timeout=60, platform=test_agent.platform, name='linux',
+                                               command='d2hvYW1p'))
+    test_ability = AbilitySchema().load(dict(ability_id='123', tactic='discovery', technique_id='auto-generated',
+                                             technique_name='auto-generated', name='Manual Command',
+                                             description='test ability',
+                                             executors=[ExecutorSchema().dump(test_executor)]))
+    test_operation.adversary.atomic_ordering.append(test_ability)
+    loop.run_until_complete(BaseService.get_service('data_svc').store(test_ability))
+    test_link = Link.load(dict(command=test_executor.command, paw=test_agent.paw, ability=test_ability,
+                               executor=test_executor, status=test_operation.link_status(), score=0, jitter=0,
+                               cleanup=0, pin=0, host=test_agent.host, deadman=False, used=[], id='456',
+                               relationships=[]))
+    test_link.host = test_agent.host
+    test_operation.chain.append(test_link)
+    loop.run_until_complete(BaseService.get_service('data_svc').store(test_operation))
+
+
+@pytest.mark.usefixtures(
+    "setup_operations_api_test"
+)
+class TestOperationsApi:
+    async def test_get_operations(self, api_client, api_cookies):
+        resp = await api_client.get('/api/v2/operations', cookies=api_cookies)
+        operations_list = await resp.json()
+        assert len(operations_list) == 1
+        operation_dict = operations_list[0]
+        assert operation_dict['name'] == 'My Test Operation'
+        assert operation_dict['id'] == '123'
```

Comment:
Let's compare with `expected_operation` (may need to make a separate fixture or save it as a variable)